uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-record-actions.html

```diff
@@ -1,14 +1,26 @@
-<div class="btn-group" uib-dropdown>
-  <button type="button" class="btn {{vm.btnClass}} dropdown-toggle" uib-dropdown-toggle>
-    <span class="visible-sm visible-md visible-lg"><i class="fa fa-plus-circle" aria-hidden="true"></i> Record New Absence</span>
-    <span class="hidden-sm hidden-md hidden-lg"><i class="fa fa-plus" aria-hidden="true"></i> New</span>
+<div class="btn-toolbar">
+  <div class="btn-group" uib-dropdown>
+    <button type="button" class="btn btn-primary btn-lg dropdown-toggle" uib-dropdown-toggle>
```

Comment:
these are buttons with icons, they have a different markup, please check the styleguide
```html
<button type="button" class="btn btn-default">
  <span class="btn-icon"><i class="fa fa-plus"></i></span> Default
</button>
```

Reply:
Done.